packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -126,15 +126,16 @@ class DebuggerScreen extends Screen {
     return false;
   }
 
+  List<CoreElement> _navEditorPanels;
+  CoreElement _sourceArea;
+  CoreElement _consoleDiv;
+
```

Comment:
nit: new lines between vars

Reply:
done. also moved to a better spot in the file.